ee-feature-pack/common/src/main/resources/modules/system/layers/base/org/jboss/as/webservices/main/module.xml

```diff
@@ -33,7 +33,7 @@
     </exports>
 
     <resources>
-        <artifact name="\${org.wildfly:wildfly-webservices-server-integration@module.jakarta.suffix@}"/>
+        <artifact name="\${org.wildfly:wildfly-webservices-server-integration}"/>
```

Comment:
Maybe it's not required, but we should remove this leading `\`.

Reply:
Fixed.